README.md

```diff
@@ -58,22 +55,30 @@ registerServiceWorker({ scope: '/' }).then((registration) => {
 Example with [Workbox Window](https://developers.google.com/web/tools/workbox/modules/workbox-window):
 
 ```js
-import {
-    Workbox
-} from 'workbox-window';
-import {
-    scriptUrl
-} from 'service-worker-loader!./sw';
+import { Workbox } from 'workbox-window';
+import { scriptUrl } from 'service-worker-loader!./sw';
 
 if ('serviceWorker' in navigator) {
-
     const wb = new Workbox(scriptUrl);
-
     wb.register();
 }
 ```
 
-### Options
+## API
+
+### Loader exports
+
+```js
+import registerServiceWorker, { scriptUrl, ServiceWorkerNoSupportError } from 'service-worker-loader!./service-worker.js';
+```
+
+* `registerServiceWorker(options)` registers the file passed through the loader as a service worker. The `options` argument is passed as the second argument to [`navigator.serviceWorker.register`](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerContainer/register)
```

Comment:
Let's add some types:
* `registerServiceWorker(options: RegistrationOptions): Promise<ServiceWorkerRegistration>`

and I think also need to add info about what it returns: https://developer.mozilla.org/ru/docs/Web/API/ServiceWorkerRegistration